templates/instances.html.j2

```diff
@@ -83,6 +115,10 @@
                                         <a class="dropdown-item" @click="deleteInstance(sm.name)" x-html="`<span class='has-text-danger'>${SVG_DELETE}</span> Delete`">
                                         </a>
                                     </div>
+                                    <div class="dropdown-content">
+                                        <a class="dropdown-item" @click="editInstance(sm)" x-html="`<span class='has-text-waring'>${SVG_EDIT}</span> Edit`">
```

Comment:
Add this `dropdown-item`  under above `dropdown-content` div itself.

Reply:
Done!